### ISSUE TITLE
Kute.js support

### ISSUE THREAD

**Issue by ershad989** [janpaepke/ScrollMagic#868]
Hi,

Scrollmagic is one of the best JS plugins ever! and I'm so happy that I got to use it. 
you guys support GSAP and it's great. but Scrollmagic is free to use while GSAP is not (for commercial usage).
If you could support kute.js which is very very like GSAP but COMPLETELY FREE as Scrollmagic that would be unforgettable.

Regards,
Ershad

**Comment by evanhessler** [comment]
I agree. Attaching my Kute.js animations to scrollmagic would be huge for me.

**Comment by janpaepke** [comment]
Better late than never on this one — thanks for the kind words about ScrollMagic.

v3 takes a fundamentally different approach to animation libraries: it is completely animation-agnostic. Instead of built-in plugins for specific libraries, v3 fires `enter`, `leave`, and `progress` events that you can wire up to any animation library — kute.js, GSAP, Motion, anime.js, or anything else. This means you get first-class support for every animation library without needing dedicated plugins. Also worth noting: GSAP went fully free/open-source in 2024.

---
**ScrollMagic v3 is now available** — a complete rewrite with an entirely new architecture and API. v2 will not receive further updates (2.0.9 is the final release).

👉 `npm install scrollmagic@next` — [Docs](https://github.com/janpaepke/ScrollMagic/blob/main/README.md) · [Release notes](https://github.com/janpaepke/ScrollMagic/releases/tag/v3.0.0-beta.0) · [npm](https://www.npmjs.com/package/scrollmagic/v/3.0.0-beta.0)
